src/commands/route/mod.rs

```diff
@@ -0,0 +1,50 @@
+extern crate serde_json;
+
+use cloudflare::endpoints::workers::{DeleteRoute, ListRoutes};
+use cloudflare::framework::apiclient::ApiClient;
+use cloudflare::framework::HttpApiClientConfig;
+
+use crate::http;
+use crate::settings::global_user::GlobalUser;
+use crate::terminal::message;
+
+pub fn list(zone_identifier: String, user: &GlobalUser) -> Result<(), failure::Error> {
+    let client = http::cf_v4_api_client(user, HttpApiClientConfig::default())?;
+
+    let result = client.request(&ListRoutes {
+        zone_identifier: &zone_identifier,
+    });
+
+    match result {
+        Ok(success) => {
+            let routes = success.result;
+            println!("{}", serde_json::to_string(&routes)?);
```

Comment:
Huh! looks like this API call doesn't use pagination in its results, which surprised me :D Printing this out as vanilla JSON should work just fine then (no need to consolidate multiple pages of results to print them out).

Reply:
is this a call to action? 😅 if so idk what it is

Reply:
no call to action! just an observation :)